setup.py

```diff
@@ -3,18 +3,8 @@
 import sys
 from setuptools import setup
 
-requires = ['six']
-tests_require = [
-    'mock;python_version<"3.4"',
-    'nose'
-]
-
-if sys.version_info.major == 2:
-    requires += ['python-dateutil>=1.0, != 2.0']
-else:
-    # Py3k
-    requires += ['python-dateutil>=2.0']
-
+requires = ['six', 'python-dateutil>=2.0']
```

Comment:
do we still need six library if we drop py2 support?

Reply:
You're right, we can remove that dependency then